lib/kitchen/driver/openstack.rb

```diff
@@ -325,11 +320,11 @@ def parse_ips(pub, priv)
 
       def add_ohai_hint(state)
         info 'Adding OpenStack hint for ohai'
-        ssh = Fog::SSH.new(*build_ssh_args(state))
-        ssh.run([
-          %(sudo mkdir -p #{Ohai::Config[:hints_path][0]}),
-          %(sudo touch #{Ohai::Config[:hints_path][0]}/openstack.json)
-        ])
+        mkdir_cmd = "sudo mkdir -p #{Ohai::Config[:hints_path][0]}"
+        touch_cmd = "sudo touch #{Ohai::Config[:hints_path][0]}/openstack.json"
+        instance.transport.connection(state).execute(
+          "#{mkdir_cmd};#{touch_cmd}"
```

Comment:
I'd use `&&` instead of `;` so the touch command isn't attempted if the mkdir fails.  It might help clean up any errors that are raised so you know why it failed (vs. the touch command being the last thing you see in STDERR).